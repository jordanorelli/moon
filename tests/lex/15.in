mix_string: this is "one" bare string that contains quotes
