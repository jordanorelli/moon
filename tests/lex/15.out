{t_name mix_string}
{t_object_separator :}
{t_string this is "one" bare string that contains quotes}
